config/always-pyzfs.m4

```diff
@@ -26,10 +26,10 @@ AC_DEFUN([ZFS_AC_CONFIG_ALWAYS_PYZFS], [
 	dnl # Require python-devel libraries
```

Comment:
In order for `--without-python` to work I additionally needed to explicitly disable `pyzfs` when there was no version of python available.

```diff
@@ -18,7 +18,13 @@ AC_DEFUN([ZFS_AC_CONFIG_ALWAYS_PYZFS], [
                        DEFINE_PYZFS='--without pyzfs'
                ])
        ], [
-               DEFINE_PYZFS=''
+               AS_IF([test $PYTHON != :], [
+                       DEFINE_PYZFS=''
+               ], [
+                       enable_pyzfs=no
+                       DEFINE_PYZFS='--without pyzfs'
+               ])
        ])
        AC_SUBST(DEFINE_PYZFS)
 

```